code/modules/cargo/bounties/science.dm

```diff
@@ -1,72 +1,72 @@
 /datum/bounty/item/science/boh
 	name = "Bag of Holding"
 	description = "Nanotrasen would make good use of high-capacity backpacks. If you have any, please ship them."
-	reward = 10000
+	reward = 5000
```

Comment:
to all this stuff: just ask sci at 12:20
or steal the service lathe

Reply:
Steal the service lathe? But yeah, Things easily made after some RnD shouldn't net 10k, while mechs requiring  materials+labor are 10k-20k

Reply:
by stealing i meant breaking in there with a bunch of bars and print the shit on your own because not even service people know the lathe exist